### PR TITLE
Spell writable correctly to match SQLiteOpenHelper.

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java [hunk 1]
```diff
@@ -153,6 +153,14 @@ public SQLiteDatabase getReadableDatabase() {
     return helper.getReadableDatabase();
   }
 
+
+  /** @deprecated Use {@link #getWritableDatabase()}. */
```
Comment (AlecStrong) [inline sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java]: why not just delete it now?

Comment (JakeWharton) [inline sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java]: it was released as public API

Comment (JakeWharton) [inline sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java]: the burdens of 1.0

Comment (AlecStrong) [inline sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java]: coolcool :+1:

**Pull request merged**